tutorials/clinical/10_ieeg_localize.py

```diff
@@ -315,6 +315,24 @@ def plot_overlay(image, compare, title, thresh=None):
                           subject='sample_ecog',
                           subjects_dir=op.join(misc_path, 'ecog'))
 
+# for ECoG, we typically want to account for "brain shift" or shrinking of the
+# brain away from the skull/dura due to changes in pressure during the
+# craniotomy
+# Note: this requires the BEM surfaces to have been computed e.g. using
+# :ref:`mne watershed_bem` or :ref:`mne flash_bem`
+raw_ecog.info = mne.preprocessing.ieeg.project_sensors_onto_brain(
```

Comment:
It would be good to show a "before" as well (with the same view/orientation etc.), so that the effect is clearly visible